src/Http/Controllers/BlogController.php

```diff
@@ -60,8 +60,6 @@ private function handleTopicRoute(Request $request, Topic $topic, ?string $slug2
     private function handlePageRoute(Request $request, Page $page, ?string $slug2 = null, ?string $slug3 = null)
     {
         if (empty($slug2)) {
-            abort_unless($page->is_leaf, 404);
```

Comment:
Removing this line (and the one below) will allow partial page paths to render.

Reply:
Thanks